### PR TITLE
Configure default OPA Blast Radius

### PR DESCRIPTION
**Pull request by simongottschlag**
This tries to add a default blast radius. Thank you @anderseknert

### CODE REVIEW

File: docker/opa-policies/test_fake_resource.rego [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          "update"
+        ]
+      },
+      "type": "fake_resource"
+    }
+  ]
+}
+
+test_create_fake_resource {
+    authz with input as input_create_fake_resource
```
Comment (anderseknert) [inline docker/opa-policies/test_fake_resource.rego:43]: Where is the `authz` rule defined? :)

Comment (simongottschlag) [reply docker/opa-policies/test_fake_resource.rego:43]: https://github.com/XenitAB/github-actions/blob/24badda863e9a963ac43e8f14d0015fd81fe8bb0/docker/opa-policies/terraform.rego#L47-L52 👍

File: docker/opa-policies/terraform.rego [hunk 2]
```diff
@@ -57,7 +61,15 @@ score = s {
             mod := crud["modify"] * num_modifies[resource_type];
             x := del + new + mod
     ]
-    s := sum(all)
+    others := [ x |
+            some resource_type
+            crud := {"delete": 100, "create": 1, "modify": 1};
```
Comment (nissessenap) [inline docker/opa-policies/terraform.rego:66]: Should we have 100 for undefined objects?
If so is it worth removing all the defined objects that doesn't have other values then default?

**Review by nissessenap — APPROVED** [review]
Minor comment with a thought but other then that LGTM

File: docker/opa-policies/terraform.rego (hunk 2 above)
Comment (simongottschlag) [reply docker/opa-policies/terraform.rego:66]: Maybe for the future

**Pull request merged**